task.go

```diff
@@ -0,0 +1,71 @@
+package mesg
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+
+	"github.com/mesg-foundation/core/api/service"
+)
+
+var errOnlyOneOutputKey = errors.New("there should be only one output key in response")
+
+// Task represents a MESG task.
+type Task struct {
+	name    string
+	handler func(*Request) Response
```

Comment:
Similar to the comment on Response type:

I think this function should return `OutputKey` and `OutputData`.
A task execution can only return one output at a time even if the task's definition declare many output.

Reply:
it now respects to service API definition as below:
```go
type defaultTask struct {
	name    Key
	handler func(*Execution) (Key, Data)
}
```